free-podcasts-screencasts-en.md

```diff
@@ -82,6 +82,7 @@
 * [Podcast] - [JavaScript Jabber](https://devchat.tv/js-jabber/)
 * [Podcast] - [The Javascript Show](http://javascriptshow.com)
 * [Podcast] - [JavaScript Air](https://javascriptair.com)
+* [Podcast] - [Frontend Five](https://frontendfive.codeschool.com/)
```

Comment:
Travis CI seems to dislike this trailing slash. I think that you should remove it ;) Also a new convention for me